config/releases.exs

```diff
@@ -43,6 +43,11 @@ kafka_endpoints =
 
 bucket_region = System.get_env("BUCKET_REGION", "local")
 
+redix_args = [
+  host: System.get_env("REDIS_HOST") || "localhost",
```

Comment:
```suggestion
  host: System.get_env("REDIS_HOST", "localhost"),
```

Elixir 1.9 added `System.get_env/3` so you don't have to do the `||` stuff anymore.